Lesson25/Anil/filter.js

```diff
@@ -0,0 +1,428 @@
+// Exercise 1:
+// Given an array of numbers, filter out the even numbers.
+const numbersExercise1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const isEven = number => number % 2 == 0;
+const evenNums = array=> [...array].filter(isEven);
```

Comment:
Filter function in JS returns a shallow copy, it does not actually modify the initial array so there's no need to copy the array before applying the filter. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
Same for all cases in this file